appveyor.yml

```diff
@@ -0,0 +1,45 @@
+version: 0.0.{build}
+image:
+  - Visual Studio 2019
+configuration: Release
+skip_commits:
+  message: /.*\[ci\-skip\].*/ 
+before_build:
+- dotnet restore all_in_one.sln
+build:
+  project: all_in_one.sln
+  verbosity: minimal
+  publish_nuget: true
+test_script:
+- dotnet test all_in_one.sln
+deploy:
+- provider: NuGet
+  api_key:
+    secure: S7SGXqFTXTOVbVLP45Ofyf/iTsIOr9o0F/QEeQkaUMqb/PkwWjXK6+CF4WAqbfWr
+  skip_symbols: true
+  artifact: /.*\.nupkg/
+  on:
+    branch: master
```

Comment:
So potentially I could use that to publish all the NuGet packages at once ? I guess I would need to change `Directory.Build.props` for every submodule. I'm not sure how the publishing to NuGet works then.

Reply:
Theoretically yes, but to be tested! You will have to change the `Directory.Build.props` on every submodule. We can put a condition on the project file to take into account another property called, for example, `GlobalVlingoVersion` that will be declared on the platform scope that can override the submodule property `VlingoVersion`.

Reply:
After long testing, it's possible what I said but I think it was a wrong idea to put a `Directory.Build.props` on the platform repository. Each repository must keep the version used to publish to keep track of the change.